Resources/doc/5-Usage-Event_Listener.md

```diff
@@ -78,6 +78,9 @@ class SitemapBlogPostSubscriber implements EventSubscriberInterface
 }
 ```
 
+**note :** you may not use this snippet as is. With large dataset, `findAll` is not a good idead. 
```

Comment:
Maybe "should" instead of "may"? It's nitpicking, really, but "may" sounds way too authoritative for something we just want to mention... ??